An error on `win-64`:
```
  C:\b\abs_f22ahcleto\croot\optree_1741694255265\_build_env\Library\bin\cmake.EXE -S C:\b\abs_f22ahcleto\croot\optree_1741694255265\work -B C:\b\abs_f22ahcleto\croot\optree_1741694255265\work\build\temp.win-amd64-cpython-311\Release -DCMAKE_BUILD_TYPE=Release -DCMAKE_LIBRARY_OUTPUT_DIRECTORY_RELEASE=C:\b\abs_f22ahcleto\croot\optree_1741694255265\work\build\lib.win-amd64-cpython-311\optree -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_RELEASE=C:\b\abs_f22ahcleto\croot\optree_1741694255265\work\build\temp.win-amd64-cpython-311\Release -DPython_EXECUTABLE=C:\b\abs_f22ahcleto\croot\optree_1741694255265\_h_env\python.exe -DPython_INCLUDE_DIR=C:\b\abs_f22ahcleto\croot\optree_1741694255265\_h_env\Include -DPython_EXTRA_INCLUDE_DIRS=C:\b\abs_f22ahcleto\croot\optree_1741694255265\_h_env\include;C:\b\abs_f22ahcleto\croot\optree_1741694255265\_h_env\Include -DPython_EXTRA_LIBRARY_DIRS=C:\b\abs_f22ahcleto\croot\optree_1741694255265\_h_env\libs;C:\b\abs_f22ahcleto\croot\optree_1741694255265\_h_env;C:\b\abs_f22ahcleto\croot\optree_1741694255265\_h_env\PCbuild\amd64 -A=x64 -Dpybind11_DIR=C:\b\abs_f22ahcleto\croot\optree_1741694255265\_h_env\Lib\site-packages\pybind11\share\cmake\pybind11

  -- Building for: Ninja

  CMake Error at CMakeLists.txt:17 (project):
    Generator
      Ninja
    does not support platform specification, but platform
      x64
    was specified.

  CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
```

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -10,7 +10,6 @@ source:
   sha256: c011c6124d6dcbfceade2d7c4f836eab66ed8cf9ab12f94535b41a71dd734637
 
 build:
-  skip: true  # [python_impl == 'pypy']
   script:
     - set CMAKE_GENERATOR=Visual Studio 16 2019 # [win]
     - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
@@ -23,7 +22,7 @@ requirements:
     - python                              # [build_platform != target_platform]
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
     - {{ compiler('cxx') }}
-    - cmake >=3.15  # to support `CMAKE_GENERATOR`
+    - cmake >=3.18  # to support `CMAKE_GENERATOR`
     - make                                # [not win]
   host:
     - pip

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -14,8 +14,8 @@ build:
   script:
     - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   script_env:           # [win]
-    # Unset `CMAKE_GENERATOR` and let Visual Studio set to its default
-    - CMAKE_GENERATOR=  # [win]
+    # Unset `CMAKE_GENERATOR_PLATFORM` and let Visual Studio set to its default
+    - set CMAKE_GENERATOR_PLATFORM=  # [win]
   missing_dso_whitelist:  # [s390x]
     - $RPATH/ld64.so.1    # [s390x]
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -10,10 +10,12 @@ source:
   sha256: c011c6124d6dcbfceade2d7c4f836eab66ed8cf9ab12f94535b41a71dd734637
 
 build:
-  script:
-    - set CMAKE_GENERATOR=Visual Studio 16 2019 # [win]
-    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script:
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  script_env:           # [win]
+    # Unset `CMAKE_GENERATOR` and let Visual Studio set to its default
+    - CMAKE_GENERATOR=  # [win]
   missing_dso_whitelist:  # [s390x]
     - $RPATH/ld64.so.1    # [s390x]
 

